README.rst

```diff
@@ -253,6 +253,13 @@ Changelog
 ========= ======================================================================
 Version   Description
 ========= ======================================================================
+0.15.0    * remove useless code (reqdme, descrption) related to old rules
```

Comment:
(reqdme, descrption) > (readme, description)

Reply:
removed the deprecated imports and fix typos